platform/backend/repository/api/src/main/kotlin/io/hamal/repository/api/Trigger.kt

```diff
@@ -65,6 +70,13 @@ interface TriggerCmdRepository : CmdRepository {
         val flowId: FlowId,
         val inputs: TriggerInputs,
         val cron: CronPattern,
+        val correlationId: CorrelationId? = null,
+        val status: TriggerStatus = TriggerStatus.Active
+    )
+
+    data class SetTriggerCmd(
```

Comment:
```suggestion
    data class SetTriggerStatusCmd
```
SetTriggerCmd - could be anything